runner_test.go

```diff
@@ -62,13 +62,32 @@ func TestScenario(t *testing.T) {
 				}
 				newBytes, err := yaml.Marshal(&scenario)
 				if assert.NoError(t, err) {
-					ioutil.WriteFile(scenarioFileName, newBytes, 0644)
+					ioutil.WriteFile(scenarioFileName, newBytes, os.ModePerm)
 				}
 			}
 
-			if scenario.Error != "" {
+			if scenario.Error != "" || scenario.ErrorStartsWith != "" {
 				require.Error(t, err)
-				require.Equal(t, replaceAbsolutePath(fmt.Sprint(err)), scenario.Error)
+				actualErr := replaceAbsolutePath(fmt.Sprint(err))
```

Comment:
You can use `|-` inside the yaml multiline string to have no trailing newline. I'll fix this in the next commit, as no error should have a trailing newline, at least not in this context, because inside https://github.com/jmattheis/goverter/blob/main/cmd/goverter/main.go#L29 every error gets a newline.

Reply:
(most errors had a double newline there, which was a bug (:.

Reply:
See https://github.com/jmattheis/goverter/commit/a0f01416a78516f24c47159d9091973766f20ce0 thanks for your work!

Reply:
neat, thanks for the nice YAML trick :). I will port your change into our fork shortly.